docs/design/feedback.rst

```diff
@@ -0,0 +1,119 @@
+.. vale off
+
+Providing effective user feedback
+=================================
+
+.. vale on
+
+When developing features, it's crucial to ensure that Users receive clear feedback and guidance when certain information or data isn't available. Instead of simply hiding tabs or displaying zeroed metrics, for example, adopt a proactive approach to inform and guide Users.
+
+Fundamental principles
+----------------------
+
+The principles of visibility, transparency, and guidance Form the foundation of an intuitive and informative User experience.
+
+- **Visibility**: keep all functionalities visible, even when there is no data available.
+- **Transparency**: communicate that information is missing.
+- **Guidance**: provide instructions on how to obtain or enable the necessary data.
+
+Visibility ensures that Users are aware of all available functionalities, even when they're not active or populated. Transparency builds trust by clearly explaining why certain information might be missing. Guidance empowers Users by providing clear paths for action and improvement. Together, these principles transform Points of frustration into solutions, helping marketing professionals complete their tasks.
+
+Practical implementation
+------------------------
+
+The practical implementation of these guidelines in Mautic goes beyond simply avoiding blank screens. It involves creating a conversation with the User, anticipating their needs, and guiding them with minimal workload. Every informative message, call to action, or configuration tip serves as a contextual mini-tutorial, educating Users about the platform's capabilities while helping them overcome obstacles.
+
+When encountering situations where data is absent, follow these guidelines:
+
+- Replace empty areas or zeroed metrics with explanatory messages. For example:
+  
+  "There is no information about the devices used yet. This happens automatically when Users interact with your Campaigns."
+  
+- Include clear Call To Actions that guide the User on how to proceed. For example:
+  
+  "No Email activity? Start sending some Campaigns to populate this data!"
+  
+- If the lack of data is due to incomplete configuration, provide direct guidance:
+  
+  "It looks like device tracking isn't enabled. Go to settings to enable it."
+  
+- Help Users understand the value of the missing data:
+  
+  "Device information helps optimize your Campaigns for different platforms. Once this data becomes available, see detailed analytics here."
+  
+- Use icons, colors, or visual elements to indicate areas that need attention.
+
+This approach not only improves immediate usability but also accelerates Users' learning curve, leading to more sophisticated use of the platform over time. Users don't feel "stuck" when encountering areas without data, but are instead motivated to explore and fill those gaps.
```

Comment:
```suggestion
This approach not only improves immediate usability but also accelerates the Users' learning curve, leading to more sophisticated use of the platform over time. Users don't feel 'stuck' when encountering areas without data, but are instead motivated to explore and fill those gaps.
```

Let's be consistent with using double quotes for speech and single quotes for everything else.